From 7bd8664c66fd72421bf3dd2cbf62d9a0eff4a037 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Mon, 1 Apr 2019 19:45:12 +0900
Subject: [PATCH 790/909] Revert "arm64: dts: r8a77990: Connect SDHI to
 IPMMU-DS1"

commit 915233498bcfac9369169af32d4b49625d61db35 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This reverts commit 2cdef56231d3098a413528456fb958d236f82937.

Multiple IPMMUs can not be operated simultaneously due to H3 Ver.1.x,
H3 Ver.2.0, M3 Ver.1.x hardware restriction.

E3 has no similar hardware restriction, but disables IPMMU for SoCs
that have hardware restriction.

As a result, This reverts commit 2cdef56231d3 ("arm64: dts: r8a77990:
Connect SDHI to IPMMU-DS1") to keep IPMMU for SDHI{0,1,3} is disabled by
default.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77990.dtsi | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a77990.dtsi b/arch/arm64/boot/dts/renesas/r8a77990.dtsi
index e5e3386b7e6e..7cfa102cefd8 100644
--- a/arch/arm64/boot/dts/renesas/r8a77990.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a77990.dtsi
@@ -1559,7 +1559,6 @@
 			max-frequency = <200000000>;
 			power-domains = <&sysc R8A77990_PD_ALWAYS_ON>;
 			resets = <&cpg 314>;
-			iommus = <&ipmmu_ds1 32>;
 			status = "disabled";
 		};
 
@@ -1572,7 +1571,6 @@
 			max-frequency = <200000000>;
 			power-domains = <&sysc R8A77990_PD_ALWAYS_ON>;
 			resets = <&cpg 313>;
-			iommus = <&ipmmu_ds1 33>;
 			status = "disabled";
 		};
 
@@ -1585,7 +1583,6 @@
 			max-frequency = <200000000>;
 			power-domains = <&sysc R8A77990_PD_ALWAYS_ON>;
 			resets = <&cpg 311>;
-			iommus = <&ipmmu_ds1 35>;
 			status = "disabled";
 		};
 
-- 
2.17.1

